src/Magnum/Math/Quaternion.h

```diff
@@ -733,6 +740,57 @@ template<class T> Matrix3x3<T> Quaternion<T>::toMatrix() const {
     };
 }
 
+/* Algorithm from:
+   https://github.com/qt/qtbase/blob/84056cdca11c63ddb9fac3d95fcfa952adaa6ea1/src/gui/math3d/qquaternion.cpp#L507 */
+template<class T> Vector3<T> Quaternion<T>::toRotation() const {
```

Comment:
Shouldn't this assert for the quaternion being normalized, similarly to how `angle()` / `axis()` have it? AFAIK Qt's quaternions are always normalized, so there they don't check this.

The docs should then mention 

> Expects the quaternion to be normalized.